### DISCUSSION TITLE
关于btoa的未定义，在5.2.0版本

### DISCUSSION THREAD

**Discussion by will11230** [ant-design/ant-design#35446] [Ideas]
<img width="1056" alt="截屏2022-05-09 下午5 42 16" src="https://user-images.githubusercontent.com/105207081/167386046-804a109c-1273-47bc-9862-d05bd92e7169.png">
这个问题在umi中的cache

**Comment by TinsFox** [comment]
```bash
yarn add swagger-ui-react
```
